### PR TITLE
NWPS-1363 - Moved breadcrumb container outside of <main> container

### PR DESCRIPTION
**Pull request by AlexKirstenZA**
@sankar-shunmuga @smanivasag I've moved the breadcrumbs as discussed, and it seems as if the styling is unaffected :+1: 

Once approved I'll cherry-pick this change into our `release/NWPS-Sprint-29` branch too.

### CODE REVIEW

File: app/views/example-template.html [hunk 1]
```diff
@@ -84,17 +84,13 @@
 {% endblock %}
 
 {% block headerEnd %}
-  <div class="site-notifications">
-  </div>
+{% endblock %}
+
+{% block pageBreadcrumbs %}
```
Comment (smanivasag) [inline app/views/example-template.html:89]: Do we need this on `app/views/example-template.html` as it seems to be inherited from `app/views/example.html` ?

File: app/views/example.html [hunk 2]
```diff
@@ -143,6 +143,9 @@
     {% block headerEnd %}
     {% endblock %}
 
+    {% block pageBreadcrumbs %}
```
Comment (smanivasag) [inline app/views/example.html:146]: Would it be good to rename the block as `breadcrumbs` as it is not within the `main` anymore? I understand that this involves updating all HTML files which are currently referencing `pageBreadcrumbs` block. Maybe worth updating it?

**Review by smanivasag — COMMENTED** [review]
@AlexKirstenZA - Thanks for the PR 👍 

Could we apply this for the prototype (`app/views/prototype.html`) as well? Thanks!

**Comment by AlexKirstenZA** [comment]
@smanivasag I actioned all the requested changes above. Thanks a lot for the review! :slightly_smiling_face:

**Comment by smanivasag** [comment]
> @smanivasag I actioned all the requested changes above. Thanks a lot for the review! 🙂

Thanks @AlexKirstenZA 👍 

Sorry, I should have mentioned this before. I think we should probably change `page__breadcrumbs` CSS class to something (e.g. `breadcrumbs`) in line with our naming convention (as it isn't inside the `<main>` section anymore).

Also, I could still some `<div class="page__breadcrumbs">...</div>` block available within `<main .../>` section for some HTML files. Could they be corrected ? Thanks!

**Comment by AlexKirstenZA** [comment]
> > @smanivasag I actioned all the requested changes above. Thanks a lot for the review! slightly_smiling_face
> 
> Thanks @AlexKirstenZA +1
> 
> Sorry, I should have mentioned this before. I think we should probably change `page__breadcrumbs` CSS class to something (e.g. `breadcrumbs`) in line with our naming convention (as it isn't inside the `<main>` section anymore).
> 
> Also, I could still some `<div class="page__breadcrumbs">...</div>` block available within `<main .../>` section for some HTML files. Could they be corrected ? Thanks!

No worries @smanivasag this has been amended now, thanks. :+1:

**Comment by smanivasag** [comment]
@AlexKirstenZA - Thanks for all the updates.

Could we get rid of the following if not required anymore?

```
File: app/assets/hee/templates/_page.scss
7: .page__breadcrumbs,
```

Thanks!

**Review by sankar-shunmuga — APPROVED** [review]
Thanks for the changes , @AlexKirstenZA . It is approved.

**Pull request merged**